docs/advanced/persistence.md

```diff
@@ -72,7 +72,7 @@ curl http://localhost:1633/pin/chunks/7b344ea68c699b0eca8bb4cfb3a77eb24f5e4e8ab5
 ```
 
 :::info
-Pinning and unpinning is possible file (as in the example), chunks, directories and bytes
+Pinning and unpinning is possible for files (as in the example), chunks, directories, and bytes
```

Comment:
i'm not exactly sure what "bytes" is, but i also haven't read the docs, so... :) i have an idea, but don't take my approval in the review as a well-informed one.